src/Features/Navigation/Menu.php

```diff
@@ -508,17 +513,22 @@ public function migrate_core_child_items( $menu ) {
 
 			// Don't add already added items.
 			$callbacks = self::get_callbacks();
-			if ( array_key_exists( $menu_item[2], $callbacks ) ) {
+			if ( array_key_exists( htmlspecialchars_decode( $menu_item[2] ), $callbacks ) ) {
+				continue;
+			}
+
+			if ( in_array( $menu_item[2], array( 'product_importer', 'product_exporter', 'product_attributes' ), true ) ) {
 				continue;
 			}
 
 			self::add_item(
```

Comment:
Can we use `add_plugin_item()` here so we can forego `menuId` and `parent` below?